src/lib.rs

```diff
@@ -60,6 +63,39 @@ pub fn inspect(config: &Config) -> Result<(), InspectError> {
 
     if config.plain {
         println!("{}", output);
+    }
+    else if config.unpretty.starts_with("flowgraph"){
+        if let Some(path) = &config.input {
+            let input_path = PathBuf::from(path);
+            let mut output_path = PathBuf::new();
+            output_path.push(input_path.file_name().unwrap());
+            output_path.set_extension("png");
+
+            // Create a temporary file to dump out the plain output
+            let tmp_file_path = tmpfile()?;
+            {
```

Comment:
Is there a reason why we have to scope this? If it's because of ownership, we could figure it out. 😉

Reply:
What can I say, I just really love useless squiggly braces.  There's no real reason why they are there aside from supporting my mental thinking model. I will remove them.